.bcr/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: "MODULE.bazel"
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+    bazel: [6.x, 7.x, 8.x]
```

Comment:
How do we keep this list updated? We use dependabot and renovate to update our Bazel version generally (see https://github.com/grpc-ecosystem/grpc-gateway/pull/5277).